lib/shopify_app/scripttags_manager.rb

```diff
@@ -2,12 +2,8 @@ module ShopifyApp
   class ScripttagsManager
     class CreationFailed < StandardError; end
 
-    def self.queue(shop_name, token)
-      ShopifyApp::ScripttagsManagerJob.perform_later(shop_name: shop_name, token: token)
-    end
-
-    def initialize(shop_name, token)
-      @shop_name, @token = shop_name, token
+    def self.queue(shop_domain)
+      ShopifyApp::ScripttagsManagerJob.perform_later(shop_domain: shop_domain)
     end
```

Comment:
I like making this object not responsible for creating the shopify session if used stand alone but I think we should still pass the token in here. Otherwise we are assuming the existence of the Shop model which may not be true. We could also use the SessionRepository object inside the job but I find that less clear than just passing in the token and letting the job create an API session.